sandbox-tools/merge-repos/src/syncReposToStaging.ts

```diff
@@ -177,84 +197,6 @@ async function removePotentialMergeConflicts(git: SimpleGit, theRepos: IRepoSync
     return null;
 }
 
-/**
```

Comment:
This function was removed as the sequence was re-worked (several times) to avoid unnecessary "merge" commits and errors that where occurring.